avaframe/ana3AIMEC/aimecTools.py

```diff
@@ -957,16 +971,22 @@ def computeRunOut(cfgSetup, rasterTransfo, resAnalysisDF, transformedRasters, si
     resAnalysisDF.loc[simRowHash, 'xRunout'] = gridx[cLower, index]
     resAnalysisDF.loc[simRowHash, 'yRunout'] = gridy[cLower, index]
     resAnalysisDF.loc[simRowHash, 'deltaSXY'] = scoord[cLower] - scoord[cUpper]
-    resAnalysisDF.loc[simRowHash, 'runoutAngle'] = np.rad2deg(np.arctan((transformedDEMRasters[cUpper, n] -
-                                                                         transformedDEMRasters[cLower, n]) /
+    #resAnalysisDF.loc[simRowHash, 'runoutAngle'] = np.rad2deg(np.arctan((transformedDEMRasters[cUpper, n] -
+    #                                                                     transformedDEMRasters[cLower, n]) /
+    #                                                                    (scoord[cLower] - scoord[cUpper])))
+    resAnalysisDF.loc[simRowHash, 'runoutAngle'] = np.rad2deg(np.arctan((zThalweg[cUpper] - zThalweg[cLower]) /
```

Comment:
Any good reason for keeping the commented lines? If not, please remove them (also below)

Reply:
removed and changed from using the elevation of the centerline of the transformedDEM to the elevation of the pathline